Experiment_Launcher_code/ExperimentManager.py

```diff
@@ -91,20 +109,24 @@ def StateActivity(self, state, mouse1, mouse2):
             print("delivering reward in the center ")
             mouse1.DeliverReward(Locations.Center, self.center_reward_time)
             mouse2.DeliverReward(Locations.Center, self.center_reward_time)
+            if self.numcompletedtrial > 0:
+
+                    self.time_to_return_to_center = time.time() - self.time_to_make_decision- self.time_start
+
 
 
         elif state == States.TrialStarted:
             Play(Sounds.Start)
-
+            self.time_start = time.time()
```

Comment:
What do we do after decision or return Abort? The next time_to_return will not be calculated accurately since we are not going through the center reward state.

Reply:
so i should put self.time_start in CenterReward?

Reply:
Actually I thought about moving the return time calculation from CenterReward to here.
I assume you ran the code with real mice who times out. Didn't you see any weird values in the return time in the logs?
Let's discuss it tomorrow

Reply:
yes youre right i will try to fix it